website/sidebars.js

```diff
@@ -329,6 +329,7 @@ const sidebars = {
               label: 'Display Data',
               items: [
                 `build-apps/how-to-guides/display-search-and-filter-table-data`,
+                `build-apps/how-to-guides/display-search-and-filter-list-data`,
```

Comment:
put this just above serverside pagination list

Reply:
Done